smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/commit/KafkaThrottledLatestProcessedCommit.java

```diff
@@ -362,7 +364,7 @@ long getLastProcessedOffset() {
         }
 
         void received(long offset) {
-            if (offset >= lastProcessedOffset) {
+            if (offset > lastProcessedOffset) {
```

Comment:
equals appears to be incorrect here. If the received offsett was already processed (equal to lastProcessedOffset) then it's already been delivered.